SUPPORT_POLICY.rst

```diff
@@ -22,16 +22,12 @@ This table describes the current support status of each major version of the AWS
       - Next status
       - Next status date
     * - 1.x
-      - Maintenance
       - End of Support
-      - 2022-07-08
+      - 
+      - 
     * - 2.x
       - General Availability
-      - Maintenance
-      - 2021-07-13
-    * - 3.x
-      -
-      - General Availability 
-      - 2021-07-13
```

Comment:
umm why are we removing this? we have a major version 3.x for the js esdk...

Reply:
We do?

Reply:
Wait... are you going to tell me that `mainline` is the branch for both v2 and v3?

Reply:
yup! https://github.com/aws/aws-encryption-sdk-javascript/blob/master/CHANGELOG.md#311-2022-03-15

Reply:
oi...

Reply:
In which case, both the 2.x and 3.x are in GA.

Reply:
Resolved by a468061